docs/v0.6.0/router.md

```diff
@@ -1,33 +1,34 @@
 # Configuring Router
 
-Router is a set of L7 load-balancing rules that can route between your services. It can add Header-based, path-based routing, cookies
-and other rules.
+Router is a set of L7 load-balancing rules that can route between your services. 
+It can add header-based routing, path-based routing, cookies, and other rules.
 
-#### Create routers
+#### Create Routers
 
-To create routers:
+To create a router:
 ```bash
 $ rio [-n $namespace] route add $name to $target
 ```
 
 Note: $name will be the router name. $target can point to individual services or a group of services.
-Target service has to be in the same namespace with router
+Target service has to be in the same namespace as the router.
 
 For example, to point to app `demo` and version `v1`:
 
 ```bash
 $ rio route add prod to demo@v1
 ```
 
-To point to all versions of app `demo`
+To point to all weighted versions of app `demo`
 
 ```bash
 $ rio route add prod to demo
 ```
+##
```

Comment:
This is ok but proper way is: 
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#hr